recipes/python-ternary/meta.yaml

```diff
@@ -0,0 +1,33 @@
+{% set version = "1.0" %}
+
+package:
+    name: python-ternary
+    version: {{ version }}
+
+source:
+    fn: python-ternary-{{ version }}.tar.gz
+    url: https://pypi.python.org/packages/source/p/python-ternary/python-ternary-{{ version }}.tar.gz
+    md5: b4afa32a9cb8b8c7156ba9f9f9b481f8
+
+build:
+    number: 0
+    script: python setup.py install
+
+requirements:
+    build:
+        - python
+    run:
+        - python
+        - matplotlib >=1.4
+
+test:
+    imports:
+        - ternary
+about:
+    home: https://github.com/marcharper/python-ternary
```

Comment:
Are you planning to contact the package owner here, or as part of the feedstock? (the latter might make more sense in this case)